live-examples/html-examples/text-content/css/dl.css

```diff
@@ -0,0 +1,14 @@
+@font-face {
+    font-family: "Fira Sans";
+    src: local("FiraSans-Regular"),
+    url("/media/fonts/FiraSans-Regular.woff2") format("woff2");
```

Comment:
Nit: Code style: Use single quotes. Here and above

Reply:
Eep. That'll take reprogramming muscle memory, as in 12 years of documenting at Mozilla I've always used double quotes.  I'll change this. :)